lib/cylc/__init__.py

```diff
@@ -19,7 +19,12 @@
 
 import os
 import sys
-from parsec import LOG
+import logging
+
+
+LOG = logging.getLogger('cylc')  # Acceptable?
+if hasattr(logging, 'NullHandler'):  # Back compat Python <2.7
```

Comment:
Can remove the back compat `if ...` test and just add the `NullHandler`.

Reply:
Sure thing! Done in 2a405368caf553563c0256d4c8552622ab6e870a